panther-labs/panther-analysis
# Add FIPS to panther-analysis and small update to README

### Background

(#1859)

### Changes

* Add a needed step to the `README`
* Add helper constants `FIPS_ENABLED` and `FIPS_SUFFIX`
* Change `boto3` connections to FIPS
* Fix type in `aws_cloudtrail_rules/aws_snapshot_made_public.py`

### Testing

* Run `panther_analysis_tool test`
* Upload python functions to dev environment, edit a rule to hit the kv-store and verify results
